x-pack/plugins/search_indices/public/components/start/elasticsearch_start.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import type { IndicesStatusResponse, UserStartPrivilegesResponse } from '../../../common';
+
+import { CreateIndexForm } from './create_index';
+
+const MAX_WIDTH = '600px';
+
+export interface ElasticsearchStartProps {
+  indicesData?: IndicesStatusResponse;
+  userPrivileges?: UserStartPrivilegesResponse;
+}
+
+export const ElasticsearchStart = ({ userPrivileges }: ElasticsearchStartProps) => {
+  return (
```

Comment:
suggestion: add a analytics event to register this flow was seen. Then can track how many see vs create an index.